app/src/main/java/com/sanjeevyadavit/magecart/model/LoginBodyRequest.kt

```diff
@@ -0,0 +1,3 @@
+package com.sanjeevyadavit.magecart.model
+
+data class LoginBodyRequest (val username: String, val password: String)
```

Comment:
Try to use gson's SerializedName here. That will help this data to not obfuscate when you enable minification in release builds.
Need to do only for data class that talks with apis.

Reply:
Thanks bro, make sense